master/buildbot/test/unit/www/test_avatar.py

```diff
@@ -663,3 +663,44 @@ def test_email_error(self):
                 'application/vnd.github.cloak-preview'})
         res = yield self.render_resource(self.rsrc, b'/?email=error@defunkt.com')
         self.assertEqual(res, dict(redirected=b'img/nobody.png'))
+
+
+class GitHubAvatarBasicAuth(TestReactorMixin, www.WwwTestMixin, unittest.TestCase):
+
+    @defer.inlineCallbacks
+    def setUp(self):
+        self.setUpTestReactor()
+
+        master = self.make_master(
+            url='http://a/b/', auth=auth.NoAuth(),
```

Comment:
Can we align to the oening parenthesis? If avatar.AvatarGithub does not fit we can extract it to a separate variable.

Reply:
Done.